js/src/personal/amanda/Amanda.main.page.tsx

```diff
@@ -0,0 +1,64 @@
+import { NavLink, Title, Text, Divider } from '@mantine/core'
+import styles from './Amanda.module.css'
+
+export function AmandaMainPage() {
```

Comment:
Might be getting a bit much with the dots in the names.
We could go for Main.page.tsx here. or Overview.page.tsx